src/sass/components/_post-box.scss

```diff
@@ -1,17 +1,21 @@
 .post-box {
   margin-bottom: 2rem;
-  
+
   .cover {
-    padding: 70% 25px 0 25px;
-    background: {
+    /* padding: 70% 25px 0 25px;
```

Comment:
Please don't push commented code. If it is:
1) necessary: it should not be commented;
2) unnecessary: we don't need it anymore;
3) unnecessary for the moment: you can always go back in Git's history and check this code out.